Mobile Buy SDK/Mobile Buy SDK Tests/test_shop_data.json

```diff
@@ -5,8 +5,8 @@
 	"channel_id": "",
 	"app_id": "",
 	"merchant_id": "",
-	"customer_email": "",
-	"customer_password": "",
+	"customer_email": "asd@asd.com",
+	"customer_password": "asdasd",
```

Comment:
I think we had these values initially but they got lost in rebase / merge conflicts (I think). How is this looking for Android @krisorr ?